internal/pkg/service/buffer/sink/tablesink/storage/repository/slice_repo_test.go

```diff
@@ -38,14 +38,16 @@ func TestRepository_Slice(t *testing.T) {
 	projectID := keboola.ProjectID(123)
 	branchKey := key.BranchKey{ProjectID: projectID, BranchID: 456}
 	sourceKey := key.SourceKey{BranchKey: branchKey, SourceID: "my-source"}
-	sinkKey := key.SinkKey{SourceKey: sourceKey, SinkID: "my-sink"}
+	sinkKey1 := key.SinkKey{SourceKey: sourceKey, SinkID: "my-sink-1"}
+	sinkKey2 := key.SinkKey{SourceKey: sourceKey, SinkID: "my-sink-2"}
+	sinkKey3 := key.SinkKey{SourceKey: sourceKey, SinkID: "my-sink-3"}
 	volumeID := storage.VolumeID("my-volume")
 	clk.Add(time.Hour)
-	fileKey1 := storage.FileKey{SinkKey: sinkKey, FileID: storage.FileID{OpenedAt: utctime.From(clk.Now())}}
+	fileKey1 := storage.FileKey{SinkKey: sinkKey1, FileID: storage.FileID{OpenedAt: utctime.From(clk.Now())}}
 	clk.Add(time.Hour)
-	fileKey2 := storage.FileKey{SinkKey: sinkKey, FileID: storage.FileID{OpenedAt: utctime.From(clk.Now())}}
+	fileKey2 := storage.FileKey{SinkKey: sinkKey2, FileID: storage.FileID{OpenedAt: utctime.From(clk.Now())}}
 	clk.Add(time.Hour)
-	fileKey3 := storage.FileKey{SinkKey: sinkKey, FileID: storage.FileID{OpenedAt: utctime.From(clk.Now())}}
+	fileKey3 := storage.FileKey{SinkKey: sinkKey3, FileID: storage.FileID{OpenedAt: utctime.From(clk.Now())}}
```

Comment:
Fixed test of the SliceRepository.

Each sink can have only one file in the `FIleWriting` state,
thus creating 3 sinks instead of one.